telebot/__init__.py

```diff
@@ -141,31 +142,31 @@ def __init__(self, token, threaded=True, skip_pending=False, num_threads=2):
         """
 
         self.token = token
-        self.update_listener = []
+        self.update_listener = list()
```

Comment:
What is the reason (here and below)? Nothing in PEP8 about it?
And here they say that it is slower: https://stackoverflow.com/questions/2972212/creating-an-empty-list-in-python

Reply:
Perhaps not really the best change. It just reads better.